include/RAJA/policy/apollo_multi/kernel/kernel_impl.hpp

```diff
@@ -29,41 +30,30 @@ struct PostLaunchKernel {
   static void exec(Apollo::Region *region, Apollo::RegionContext *context)
   {
     region->end(context);
-    //std::cout << "PostLaunch *NOT* CudaKernelAsync\n";
   }
 };
 
-
+#if defined(RAJA_ENABLE_CUDA)
 template<typename... Args>
 struct PreLaunchKernel<RAJA::statement::CudaKernelAsync<Args...>> {
   static void exec(Apollo::Region *region, Apollo::RegionContext *context)
   {
-    using callback_t = RAJA::apollo::ApolloCallbackDataPool::callback_t;
-    callback_t *cbdata =
-        reinterpret_cast<callback_t *>(region->callback_pool->get());
-    context->isDoneCallback =
-        RAJA::apollo::ApolloCallbackHelper::isDoneCallback;
-    context->callback_arg = cbdata;
-
-    //std::cout << "PreLaunch CudaKernelAsync\n";
-    // TODO: how to handle multiple streams?
-    cudaEventRecord(cbdata->start, 0);
+    context->timer = Apollo::Timer::create<Apollo::Timer::CudaAsync>();
```

Comment:
Timer is not currently under the Apollo name (Apollo::Timer -> Timer). Do you think it's better to include it?

Reply:
I think it should be in the Apollo namespace for now.

Reply:
Makes sense. I made the change to Apollo to match what RAJA expects.